src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -92,4 +109,80 @@ export class BitcoinAdapter extends BaseProtocolAdapter {
     }
     return lastNotEmptyIdx;
   }
+
+  static async spend(
```

Comment:
We should not use any static methods and props within the adapters as there is no way of enforcing the typing of them - the the `BaseProtocolAdapter` does not have the `spend` method.

Reply:
Defined `spend` method under `BaseProtocolAdapter`. Please have a look.